proposals/103:ymax-alpha2/test/ymax.test.js

```diff
@@ -0,0 +1,70 @@
+// @ts-check
+import '@endo/init/legacy.js'; // compat with tendermint-rpc < 0.34 which has axios
+import test from 'ava';
+
+import { boardSlottingMarshaller } from '@agoric/client-utils';
+import { executeOffer, GOV1ADDR, queryVstorage } from '@agoric/synthetic-chain';
+import { Far } from '@endo/far';
+import { passStyleOf } from '@endo/pass-style';
+
+/**
+ * @import {OfferSpec} from '@agoric/smart-wallet/src/offers.js';
+ */
+
+const oldBoardId = 'board010155'; // from A3P proposal 100
+
+const getCellValues = ({ value }) => {
+  return JSON.parse(value).values;
+};
+
+test('ymax in agoricNames failed to update', async t => {
+  const instancePath = 'published.agoricNames.instance';
+  const instanceRaw = await queryVstorage(instancePath);
+  const capData = JSON.parse(getCellValues(instanceRaw).at(-1));
+
+  const m = boardSlottingMarshaller((slot, iface) => {
+    return Far('SlotReference', {
+      getDetails: () => ({ slot, iface }),
+    });
+  });
+  const instances = Object.fromEntries(m.fromCapData(capData));
+  const { ymax0 } = instances;
+
+  t.is(passStyleOf(ymax0), 'remotable');
+  const { slot, iface } = ymax0.getDetails();
+  // The proposal was meant to update this, but it didn't because the CoreEval failed in Mainnet.
+  // Verify that the A3P history matches Mainnet
+  t.is(slot, oldBoardId);
+});
+
+test('sending offer to ymax0 contract results in error', async t => {
+  /** @type {OfferSpec} */
+  const offer = {
+    id: 'test-ymax-offer',
+    invitationSpec: {
+      source: 'agoricContract',
+      instancePath: ['ymax0'],
+      callPipe: [['makeInvitation']],
```

Comment:
This method name would  result in an error. Checking for `vat terminated` below addresses it, but it would be a bit cleaner to use...

```suggestion
      callPipe: [['makeOpenPortfolioInvitation']],
```